README.md

```diff
@@ -1,44 +1,110 @@
+
+**It's still a work in progress**
+
+
+## Installing
 
-Images uploaded by users are cropped with easy-thumbnails
-to improve the site performance.
+### Requirements
 
-It's still a work in progress
+* [Python](https://python.org) 3.4 or newer
+* [PostgreSQL](https://www.postgresql.org) running with a _database_, _username_ and _password_ to be used with Pets.
+
+
+### Fork and clone the repository
+
+First fork the project using GitHub, than clone it locally:
+
+```console
+git clone https://github.com/<username>/pets.git
+cd pets
+```
 
+### Configure your instance
 
-### How to contribute to the project
+The project configuration uses [python-decouple](https://pypi.python.org/pypi/python-decouple/) to dynamically read environment variables and `.env` files.
 
-1. Fork and clone the repository;
-2. Configure your instance (python-decouple);
-   * Remember to configure the username, password and database in your postgreSQL. 
-3. Install Pillow dependencies;
-4. Install PhantonJS, see the julionc [tutorial](https://gist.github.com/julionc/7476620);
-5. Install project requirements;
-6. Execute all tests, it will take some minutes. 
+If you want, you can get startted by copying `contrib/sample-env` as `.env`:
 
 ```console
-git clone https://github.com/<username>/pets.git && cd pets
 cp contrib/sample-env pets/.env
+```
+
+Then you have to set following variables.
+
+#### Basic Django settings
+
+* **`SECRET_KEY`**: [Django's secret key](https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-SECRET_KEY)
+* **`ALLOWED_HOSTS`** (e.g. `127.0.0.1, .localhost`) [Django's allowed hosts](https://docs.djangoproject.com/en/1.9/ref/settings/#allowed-hosts)
+* **`DJANGO_SETTINGS_MODULE`**: In order to make development and deploy to production simpler there's two settings module; `pets.settings.dev` for development and `pets.settings.dev` for production.
+
+#### Database
+
+* **`DB_NAME`**, **`DB_USERNAME`**, **`DB_PASSWORD`**, **`DB_IP`**, **`DB_PORT`**: Credentials to access Pets's database.
+* **`DB_ENGINE`**: Django databse backend to connect to Pets's database (e.g. `django.db.backends.postgresql_psycopg2`)
+* **`DB_CONN_MAX_AGE`**: (e.g. `0`) [Django's database `CONN_MAX_AGE`](https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-CONN_MAX_AGE`)
+
+#### Mail server for sending e-mails
+
+* **`EMAIL_PORT`**, **`EMAIL_HOST`**, **`EMAIL_HOST_PASSWORD`** and **`EMAIL_HOST_USER`**: SMTP credentials to a mail server.
+
+#### OAuth
+
+If you want to login via social media, you will have to create apps as a developer [at Facebook](https://developers.facebook.com) and/or [Twitter](https://apps.twitter.com). Once you're done, set the _app secret_ and _app key_ for each of them:
+
+* `SOCIAL_AUTH_FACEBOOK_KEY`
+* `SOCIAL_AUTH_FACEBOOK_SECRET`
+* `SOCIAL_AUTH_TWITTER_KEY`
+* `SOCIAL_AUTH_TWITTER_SECRET`
+
+While setting you app there you should add the proper redirect URLs.
```

Comment:
To be honest, I'm not sure what are you talking about. :laughing: 
If it's about the "callback", I'm using the default provided by `python-social-auth`, for reference, [here](http://psa.matiasaguirre.net/docs/configuration/settings.html#urls-options)

Maybe we could remove remove this line altogether, what do you think?